File: Backend/SportMatchAPI/SportMatchAPI/DTO/LocationQueryDTO.cs
namespace SportMatchAPI.DTO
{
    public class LocationQueryDTO
    {
        public double Latitude { get; set; }
        public double Longitude { get; set; }
        public double RadiusInKm { get; set; } = 5; // Mặc định quét bán kính 5km
    }
}

File: Frontend/SportMatch2/app/src/main/java/com/example/sportmatch/data/dto/LocationQueryDto.kt
package com.example.sportmatch.data.dto

import com.google.gson.annotations.SerializedName

data class LocationQueryDto(
    @SerializedName("latitude") val latitude: Double,
    @SerializedName("longitude") val longitude: Double,
    @SerializedName("radiusInKm") val radiusInKm: Double = 5.0,
    @SerializedName("sportType") val sportType: String? = null,
    @SerializedName("filterDate") val filterDate: String? = null
)